flink-python/pyflink/table/tests/test_environment_completeness.py

```diff
@@ -38,8 +38,8 @@ def excluded_methods(cls):
         # registerFunction and listUserDefinedFunctions should be supported when UDFs supported.
         # registerExternalCatalog, getRegisteredExternalCatalog and listTables
         # should be supported when catalog supported in python.
-        return {'registerExternalCatalog', 'getRegisteredExternalCatalog',
-                'registerFunction', 'listUserDefinedFunctions', 'listTables'}
+        return {'registerExternalCatalog', 'getRegisteredExternalCatalog', 'connect',
+                'registerFunction', 'listUserDefinedFunctions', 'listTables', 'getCompletionHints'}
```

Comment:
Can you add the comment for why we excluded the `getCompletionHints`?
such as `Will be removed in the next release`. What do you think?

Reply:
That makes sense. I have added comment here now.